### PR TITLE
doc/releases: add release notes for mimic v13.2.7

### PR DESCRIPTION
**Pull request by smithfarm**

### CODE REVIEW

File: doc/releases/mimic.rst [hunk 1]
```diff
@@ -1,3 +1,374 @@
...
+  a percentage of "osd_heartbeat_grace" to determine the threshold.  A value of
+  zero disables the warning. A new configuration option "mon_warn_on_slow_ping_time",
+  specified in milliseconds, overrides the computed value, causing a warning
+  when OSD heartbeat pings take longer than the specified amount.  A new admin
+  command "ceph daemon mgr.# dump_osd_network [threshold]" lists all connections
+  with a ping time longer than the specified threshold or value determined by
+  the config options, for the average for any of the 3 intervals.  A new admin
+  command ceph daemon osd.# dump_osd_network [threshold]" does the same but only
+  including heartbeats initiated by the specified OSD.
+
```
Comment (neha-ojha) [inline doc/releases/mimic.rst]: Can we include the following?
```suggestion
* "osd_deep_scrub_large_omap_object_key_threshold" has been lowered to detect an object with large number of omap keys more easily.
```

Comment (smithfarm) [reply doc/releases/mimic.rst]: @neha-ojha It's the default value of that parameter that has been lowered, right? Added that - please take a look.

Comment (neha-ojha) [reply doc/releases/mimic.rst]: looks good, thanks!

**Comment by smithfarm** [comment]
Changelog:

* found that two of the PRs merged for v13.2.7 were reverted, so deleted four lines from the changelog: two lines for the original backport PRs plus two lines for the revert PRs

**Comment by smithfarm** [comment]
jenkins render docs

**Pull request merged**